### PR TITLE
WSTEAMA 1111 add facebook meta data to homepages and frontpages

### PR DESCRIPTION
**Pull request by holchris**
Resolves JIRA 1111 https://jira.dev.bbc.co.uk/browse/WSTEAMA-1111

Overall changes
======
Add meta property to all WS languages front and home pages to allow publishing via Social Flow

Code changes
======

Testing
======
- unit test added

### CODE REVIEW

File: src/app/components/Metadata/index.test.tsx [hunk 1]
```diff
@@ -454,6 +454,18 @@ it('should render the facebook metatags', async () => {
   });
 });
 
+it('should render the facebook domain verification', async () => {
+  render(<CanonicalNewsInternationalOrigin />);
+
+  await waitFor(() => {
+    const fbAppId = document
```
Comment (karinathomasbbc) [inline src/app/components/Metadata/index.test.tsx]: Can we rename this variable to something like `facebookDomainVerification`?

**Review by karinathomasbbc — APPROVED** [review]

File: src/app/components/Metadata/index.test.tsx [hunk 2]
```diff
@@ -128,6 +132,19 @@ const CanonicalNewsInternationalOrigin = (
...
+    bbcOrigin={dotComOrigin}
+    platform="canonical"
+    id="c0000000001o"
+    pageType={ARTICLE_PAGE}
+    pathname="/news/articles/c0000000001o"
+    {...newsArticleMetadataProps}
+    {...props}
+  />
+);
+
```
Comment (karinathomasbbc) [inline src/app/components/Metadata/index.test.tsx]: 👋 I didn't mean for you to add this - what you had previously was fine. I only meant to rename the variable name in the test... 😬

File: src/app/components/Metadata/index.test.tsx (hunk 1 above)
Comment (karinathomasbbc) [reply src/app/components/Metadata/index.test.tsx]:
```suggestion
    const facebookDomainVerification = document
```

Comment (karinathomasbbc) [reply src/app/components/Metadata/index.test.tsx]: This was the only change I expected - not adding a new type / interface for this. Could you please revert the previous commit, and just make this suggested change above?

**Review by karinathomasbbc — CHANGES_REQUESTED** [review]

**Comment by amoore108** [comment]
Just to check that it's ok that this will be applied to all page types that use this, not just home and front pages?

File: src/app/components/Metadata/index.test.tsx (hunk 2 above)
Comment (holchris) [reply src/app/components/Metadata/index.test.tsx]: My bad! Sorry I boiled the ocean there!

Comment (karinathomasbbc) [reply src/app/components/Metadata/index.test.tsx]: No worries. I think the comments / conversation view can be confusing if you view it on the PR landing page. Instead, if you look at the comments within "Files Changed" you can see more clearly what my intent was: https://github.com/bbc/simorgh/pull/11409/files - you can also filter on unresolved conversations / view one at a time, to make sure you have considerered everything. Let's have a look at this in more detail next week.

**Comment by karinathomasbbc** [comment]
> Just to check that it's ok that this will be applied to all page types that use this, not just home and front pages?

Eek that is a good question. According to https://developers.facebook.com/docs/sharing/domain-verification/verifying-your-domain it (only? bare minimum?) needs to be added to your domain home page... I wonder if for now we could proceed with this PR as it stands, as editorial are unable to use social flow, and we would like to let them resume using it ASAP. 

We could take this question away and do a bit more of an investigation, and consider only adding the meta tag on Home and Front Pages only?

@holchris could we add a subtask to the story to look into this please?

**Review by karinathomasbbc — APPROVED** [review]

**Comment by amoore108** [comment]
> > Just to check that it's ok that this will be applied to all page types that use this, not just home and front pages?
> 
> 
> 
> Eek that is a good question. According to https://developers.facebook.com/docs/sharing/domain-verification/verifying-your-domain it (only? bare minimum?) needs to be added to your domain home page... I wonder if for now we could proceed with this PR as it stands, as editorial are unable to use social flow, and we would like to let them resume using it ASAP. 
> 
> 
> 
> We could take this question away and do a bit more of an investigation, and consider only adding the meta tag on Home and Front Pages only?
> 
> 
> 
> @holchris could we add a subtask to the story to look into this please?

Yea that's grand, can't see it affecting much if it's on all page types personally, but good to check after this goes in 👍🏻

**Review by amoore108 — APPROVED** [review]

**Comment by holchris** [comment]
@karinathomasbbc and @amoore108 I will do, thanks for highlighting this!

**Pull request merged**